templates/learn.html

```diff
@@ -24,7 +24,7 @@
         Bevy Rust API Docs
       </div>
       <div class="card-description">
-        Learn about Bevy's Rust API using the generated Rust documentation.
+Learn how to use Bevy's types, traits and methods using the in-depth reference documentation, complete with inline examples.
```

Comment:
could you indent this line?

Reply:
Not sure why it suddenly gone unindented, will do.